week-1-ts-ecosystem/w01e05-my-own-cli/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "w01e05-my-own-cli",
+  "version": "1.0.0",
+  "description": "_~ \"Aplikacja do handlu\" strikes back 😱_",
+  "main": "src/prepare.ts",
+  "type": "module",
+  "scripts": {
+    "demo:prepare": "ts-node-esm src/prepare.ts",
```

Comment:
Coś co mnie najbardziej razi to te rozszerzenia .mts do których jesteś zmuszony przez narzędzie.

Okazuje się że jeśli zapiszesz:
```
ts-node --esm --experimentalSpecifierResolution node
```

To bedzie działać dla rozszerzeń `.ts`

Jednak to też - nie w tym rzecz. Bo to przygotowanie pod `dev`
Brakuje mi tutaj skryptów budujących `prod`.

Czyli prostego `tsc`.

Generalnie ładnie zacząłeś to zadanie.

Poczytaj o:
`npm link --save` żeby sobie podliknować typy z: `w01e03`

Ew. dla `dev` pokombinuj z narzędziem:
[`tsc-watch`](https://www.npmjs.com/package/tsc-watch)

Ma dużo dużo przyjemniejszy setup i bije tutaj `ts-node` na głowę.

Reply:
Powinien być przygotowane config pod produkcję, tak żeby po wybudowaniu tego dostać pliki `.js` i to one finalnie muszą działać pod Node :)